src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncSpillTests.cs

```diff
@@ -6418,5 +6418,41 @@ static async Task Main()
             var verifier = CompileAndVerify(source, expectedOutput: "1");
             verifier.VerifyDiagnostics();
         }
+
+        [Fact, WorkItem(48255, "https://github.com/dotnet/roslyn/issues/48255")]
+        public void MakeRefWithStackalloc()
+        {
+            var source = @"
+using System;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        var ts = new TestStruct();
+        Console.WriteLine(ts.ToString());
+    }
+  }
+
+unsafe struct TestStruct
+{
+    public override string ToString()
+    {
+        Span<char> chars = stackalloc char[4];
+        var reference = __makeref(this);
```

Comment:
>var reference = __makeref(this); [](start = 8, length = 32)

The scenario looks too simple. I think we would want to add a test that actually makes the spiller to do some real work and, perhaps, actually do some spilling based on ```_makeref``` argument.

Reply:
@AlekseyTs As mentioned in the PR description, I don't actually know what does the spiller do. Hence, I don't know how to update the test to make the spiller do real work.